doc/source/management/data_at_rest_encryption.rst

```diff
@@ -417,35 +412,14 @@ Dumping of encrypted binary logs involves decryption, and can be done using
    relevant to binary logs, encrypted relay logs can not be dumped/decrypted
    in this way.
 
-.. rubric:: Upgrading from |Percona Server| |changed-version| to any higher version
 
-The key format in the :ref:`keyring vault plugin
-<keyring_vault_plugin>` was changed for binlog encryption in
-|Percona Server| |changed-version| release. When you are upgrading from
-|Percona Server| 5.7.20-19 to any version in |Percona Server| 5.7
-or 8.0 series, the binary log encryption will work after you complete the following steps:
+.. rubric:: Upgrading from |Percona Server| |changed-version| to any higher version
 
-- Upgrade to a version higher than |Percona Server| |changed-version|
-- Start the server without enabling the binary log encryption: :bash:`--encrypt_binlog=OFF`
-- Enforce the the key rotation: :mysql:`SELECT rotate_system_key("percona_binlog")`
-- Restart the server enabling the binary log encryption: :bash:`--encrypt_binlog=OFF`
+.. include:: ../.res/text/encrypt_binlog.removing.txt
 
 .. |changed-version| replace:: 5.7.20-19
 
 
-System Variables
-----------------
-
-.. variable:: encrypt_binlog
-
-  :cli: ``--encrypt-binlog``
-  :dyn: No
-  :scope: Global
-  :vartype: Boolean
-  :default: ``OFF``
-
-The variable turns on binary and relay logs encryption.
-
 Temporary file encryption
```

Comment:
we should leave the description, just add that :version-info: that it was removed in 8.0.15.

Reply:
Ready for review

Reply:
why not link to :rn:`8.0.15-5`" as that would link to the release notes?